Index: a/drivers/hid/hid-multitouch.c
===================================================================
--- a/drivers/hid/hid-multitouch.c	2016-07-04 08:14:33.866466000 +0000
+++ b/drivers/hid/hid-multitouch.c	2016-07-04 08:14:59.330466000 +0000
@@ -395,11 +395,6 @@
 			td->is_buttonpad = true;
 
 		break;
-	case 0xff0000c5:
-		/* Retrieve the Win8 blob once to enable some devices */
-		if (usage->usage_index == 0)
-			mt_get_feature(hdev, field->report);
-		break;
 	}
 }
 
