From 324de04c8ae0132700eb22cd777127c6445294b0 Mon Sep 17 00:00:00 2001
From: Changhyeok Bae <changhyeok.bae@lge.com>
Date: Sat, 7 Jan 2017 14:33:39 +0900
Subject: [PATCH 1/2] Correct label on RPI

---
 src/controls/Label.qml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/controls/Label.qml b/src/controls/Label.qml
index f9bc8cd..5909807 100644
--- a/src/controls/Label.qml
+++ b/src/controls/Label.qml
@@ -86,7 +86,6 @@ Text {
     id: label
     color: SystemPaletteSingleton.windowText(enabled)
     activeFocusOnTab: false
-    renderType: Settings.isMobile ? Text.QtRendering : Text.NativeRendering
     Accessible.name: text
     Accessible.role: Accessible.StaticText
 }
-- 
1.9.1

