From 6e399190d0be3eaf7ab4b508a758923e0a2de123 Mon Sep 17 00:00:00 2001
From: Changhyeok Bae <changhyeok.bae@lge.com>
Date: Sat, 7 Jan 2017 14:35:11 +0900
Subject: [PATCH 2/2] Correct textfield on RPI

---
 src/controls/TextField.qml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/controls/TextField.qml b/src/controls/TextField.qml
index d6b9b9c..263fd83 100644
--- a/src/controls/TextField.qml
+++ b/src/controls/TextField.qml
@@ -632,7 +632,6 @@ Control {
         color: __panel ? __panel.placeholderTextColor : "darkgray"
         clip: contentWidth > width;
         elide: Text.ElideRight
-        renderType: __style ? __style.renderType : Text.NativeRendering
     }
 
     TextInputWithHandles {
@@ -659,8 +658,6 @@ Control {
         color: __panel ? __panel.textColor : "darkgray"
         clip: contentWidth > width
 
-        renderType: __style ? __style.renderType : Text.NativeRendering
-
         Keys.forwardTo: textfield
 
         EnterKey.type: control.EnterKey.type
-- 
1.9.1

